CHANGELOG.md

```diff
@@ -1,4 +1,29 @@
-# Changelog
```

Comment:
heading got removed ?

Reply:
Other repos don't have this so making all changelog similar